### PR TITLE
JR2-468 Job publication edit view component

### PR DESCRIPTION
**Pull request by Invizible**

### CODE REVIEW

File: src/main/webapp/app/shared/job-publication/job-publication.model.ts [hunk 1]
```diff
@@ -98,17 +106,25 @@ export enum Status {
     UNKNOWN
 }
 
+export enum Locale {
+    DE,
+    FR,
+    IT,
+    EN
```
Comment (pellenberger) [inline src/main/webapp/app/shared/job-publication/job-publication.model.ts]: **EN** is not a valid Locale to send to the API. If the locale is set to **en** in the frontend, send **DE** to the API.

File: src/main/webapp/i18n/de/home.json [hunk 2]
```diff
@@ -101,7 +101,32 @@
                 },
                 "reset": "Zur\u00fccksetzen",
                 "submit": "Job publizieren",
-                "workload": "Arbeitspensum"
+                "submit.success": "Job publication saved!",
+                "submit.error": "Job publication could not be saved!",
+                "workload": "Arbeitspensum",
+                "education-level": {
+                    "PRIMAR_OBLIGATORISCHE_SCHULE": "PRIMAR OBLIGATORISCHE SCHULE",
```
Comment (pellenberger) [inline src/main/webapp/i18n/de/home.json]: I sent you the translations in DE, FR and IT

Comment (pellenberger) [reply src/main/webapp/i18n/de/home.json]: Same for education levels.

Comment (Invizible) [reply src/main/webapp/i18n/de/home.json]: Ok, will update

File: src/main/webapp/app/shared/job-publication/job-publication.model.ts (hunk 1 above)
Comment (Invizible) [reply src/main/webapp/app/shared/job-publication/job-publication.model.ts]: will remove

**Pull request merged**